templates/kernel/lifecycle.html

```diff
@@ -0,0 +1,812 @@
+{% extends "kernel/base_kernel.html" %}
+
+{% block title %}Ubuntu kernel lifecycle and enablement stack{% endblock %}
+
+{% block meta_description %}Canonical provides long-term support (LTS) kernels for Ubuntu LTS releases. Canonical also provides interim operating system releases with updated kernels every 6 months. Learn about the lifecycle of each kernel through the lifespan of support.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1d01SFs7c56ge2o92g6Rwr3SjG8rtxm3jNwIyL1N8mQg/edit#heading=h.929qtgja1pr8{% endblock %}
+
+{% block content %}
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h1>
+        Ubuntu kernel lifecycle and enablement stack
+      </h1>
+      <p>
+        The Ubuntu LTS enablement also called HWE or Hardware Enablement, stacks provide the newer kernel and X support for existing Ubuntu LTS releases. These stacks can be installed manually, but are also available when installing with Ubuntu LTS point release media.
+      </p>
+      <p>
+        These newer enablement stacks are meant for desktop and server and even recommended for cloud or virtual images. However, if you want to remain on the original GA (General Availability) stacks, the options are:
+      </p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">
+          Install from a previous 12.04.0/12.04.1/14.04.0/14.04.1/16.04.0/16.04.1/18.04.0/18.04.1 point release and update. Previous releases are archived at <a class="" href="http://old-releases.ubuntu.com/">old-releases.ubuntu.com</a>
+        </li>
+        <li class="p-list__item is-ticked">
+          Perform an update or upgrade to an LTS release from a previous release
+        </li>
+        <li class="p-list__item is-ticked">
+          Perform a network install using the netboot images rather than the new &lt;release&gt;-netboot images
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>
+        Installation
+      </h2>
+
+      Show instructions for:
+      <a tabindex="1" class="p-menu-18-04">18.04 LTS</a> -
```

Comment:
Can I suggest you use the tab component? It's unclear what is going to happen when selecting an option. https://docs.vanillaframework.io/patterns/tabs